recipe/build.sh

```diff
@@ -72,20 +71,25 @@ elif [[ "${target_platform}" == linux-* ]]; then
     sed -i 's/--threads//g' $f
     sed -i 's/-fuse-ld=gold//g' $f
     sed -i 's/--thread-count=4//g' $f
-    if [[ "$f" != "out.gn/obj/v8_libbase.ninja" ]]; then
+    if [[ "$f" != "out.gn/obj/v8_libbase.ninja" ]]; then
```

Comment:
So... somehow a non-breaking space slipped into the build script before.
Fixing, i.e., replacing by a normal space, caused the linking to fail (hence the `LDFLAGS` addition below.

Reply:
Whilst writing the above, I noticed that the `for`/`if/ code is broken..

Reply:
(NB: I also have no idea why the script didn't fail with the non-breaking space. I would have expected an `[[ ...: command not found`.)

Reply:
(Ah, just overlooked it before: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=445053&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=836 . And since it's in an `if` statement it went through without stopping the build.)